gallery/src/styles/NxCard/NxCardPage.tsx

```diff
@@ -103,18 +98,37 @@ const NxCardPage = () =>
               <code className="nx-code">.nx-card__content</code> applied to a
               {' '}<code className="nx-code">&lt;div&gt;</code> wraps
               {' '}<code className="nx-code">.nx-card__call-out</code> and
-              {' '}<code className="nx-code">.nx-card__call-out-text</code> in the alternate row card layout.
+              {' '}<code className="nx-code">.nx-card__text</code> or any other card content.
             </td>
           </tr>
           <tr className="nx-table-row">
-            <td className="nx-cell"><code className="nx-code">nx-icon--card-call-out</code></td>
+            <td className="nx-cell"><code className="nx-code">.nx-card__call-out</code></td>
+            <td className="nx-cell">
+              Element which contains a very small piece of content, frequently graphical, meant for visual impact
```

Comment:
Much of the content in this cell seems like it should be in the details cell instead.

Reply:
> Much of the content in this cell seems like it should be in the details cell instead.

I like it better with minimal text, highlights that the SCSS is effecting the width, not the content: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/38859601e2652ef2c8fd2d2d283de7d9654cafb2

Reply:
> I like it better with minimal text, highlights that the SCSS is effecting the width, not the content: 3885960

That commit isn't relevant.  This comment is about the documentation table in the page

Reply:
> > I like it better with minimal text, highlights that the SCSS is effecting the width, not the content: [3885960](https://github.com/sonatype/sonatype-react-shared-components/commit/38859601e2652ef2c8fd2d2d283de7d9654cafb2)
> 
> That commit isn't relevant. This comment is about the documentation table in the page

oops: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/21c7cf5d9c42a40dd9fbedd90ddaa78bd3ffaa3b